syntax = "proto2";
package rare.protos;

import "rare/protos/rnn_cell.proto";

message DecoderConfig {
  oneof decoder_oneof {
    AttentionDecoder attention_decoder = 1;
  }
}

message AttentionDecoder {
  optional RnnCell rnn_cell = 1;
  optional uint32 num_attention_units = 2 [default=128];
  optional uint32 attention_conv_kernel_size = 3 [default=5]; 
}
